CHANGELOG.md

```diff
@@ -1,7 +1,9 @@
+# Changelog
```

Comment:
I will test if we can add a header, as previously did not pass the pub.dev changelog validation